Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -106,6 +115,15 @@
             "helpMarkDown": "Specify a name for the output variable that is generated for the URL of the Azure App Service. The variable can be consumed in subsequent tasks."
         },
         {
+            "name": "TakeAppOfflineFlag",
```

Comment:
We moved this up as it is option is available irrespective of web deploy or not. When we enable web deploy check box we wanted to keep the options which are specific to that only below that check box. Otherwise this is getting mixed with that and hence we (me and Roopesh) moved it up.